src_new/controllers/gpt.js

```diff
@@ -833,7 +829,16 @@ function addHooksIfPossible(win) { // TDD, i/o : done
                     var theObject = window.pbjs;
                     var functionName = "addAdUnits"
                     util.addHookOnFunction(theObject, false, functionName, refThis.newAddAdUnitFunction);
+                    if(vdetails = pbjs.version.split('.') && vdetails.length===3 && vdetails[0].includes("v3") && +vdetails[1] >= 3){
```

Comment:
avoid assignment in if condition, Go syntax :P

Reply:
Done